examples/bch-example/src/app/bch.virtualAccount.example.ts

```diff
@@ -35,6 +35,23 @@ export async function bchVirtualAccountExample() {
   // Fund your address here: https://faucet.fullstack.cash/
   console.log(`Fund me ${address.address} to send virtual account transaction!`)
 
+  // If you have funds on account - you can transfer it to another bch address
+  // You can find more details in https://apidoc.tatum.io/tag/Blockchain-operations#operation/BchTransfer
+  const result = await bchSDK.virtualAccount.send({
+    senderAccountId: virtualAccount.id,
+    address: 'xxxxxxxxx',
+    amount: '1',
+    keyPair: [
```

Comment:
use example iwth xpub + mnemonic and no attr

Reply:
isn't the example bellow the xpub + mnemonic and no attr? is that one correct?